static_test_driver/static_test_driver.ino

```diff
@@ -235,40 +222,32 @@ void loop() {
   
   BEGIN_READ
   READ_FLAG(zero_force) {
-    #if CONFIGURATION == MK_2
     Serial.println(F("Zeroing load cell"));
     scale.tare(); // Load Cell, Assuming there is no weight on the scale, reset to 0
-
-    #else
-    if (force_zero_ready) {
-      Serial.println(F("Zeroing force plate"));
-      force_zero_ready = false;
-      force_zero_val = 0;
-      for (int i = 0; i < FORCE_PLATE_NUM_HIST_VALS; i++)
-        force_zero_val += force_hist_vals[i];
-      force_zero_val /= FORCE_PLATE_NUM_HIST_VALS;
-    }
-    else {
-      Serial.println(F("Force zero value not ready"));
-    }
-
-    #endif
   }
-  #if CONFIGURATION == MK_2
-  READ_FLAG(zero_pressure) {
+  
+  //TODO: Update flag names to match the following code 
```

Comment:
Lucas you never answered this question, does this need to be done at all or is the flag naming correct?